src/vs/platform/audioCues/browser/audioCueService.ts

```diff
@@ -268,8 +268,8 @@ class Cache<TArg, TValue> {
 	constructor(private readonly getValue: (value: TArg) => TValue) {
 	}
 
-	private _getKey(event: IAudioCueEvent): string {
-		return `${event.cue.name}-${event.userGesture}`;
+	private _getKey(arg: TArg): string {
+		return JSON.stringify(arg);
```

Comment:
Also, `TArg` in the map would just be `unknown` then.

Reply:
yes, just did this. but not sure how to avoid this cast https://github.com/microsoft/vscode/blob/93b603aada88cbf60731adf296ab114736695085/src/vs/platform/audioCues/browser/audioCueService.ts#L279

Reply:
figured it out